app/assets/javascripts/timeout/timeoutDialog.js

```diff
@@ -196,15 +196,14 @@ function timeoutDialog(options) {
   function setupDialog() {
     var $countdownElement = $('<span id="timeout-countdown" class="countdown">');
     var $element = $('<div>')
-      .append(settings.title ? $('<h1 class="govuk-heading-m push--top">').text(settings.title) : '')
-      .append($('<p class="govuk-body" id="timeout-message" role="text">').text(settings.message + ' ')
+      .append($('<p class="govuk-body" id="timeout-message" role="text">').text(settings.message + '  ')
         .append($countdownElement)
         .append('.'))
       .append($('<button id="timeout-keep-signin-btn" class="govuk-button govuk-!-margin-right-1">').text(settings.keepAliveButtonText))
-      .append($('<button id="timeout-sign-out-btn" class="govuk-button govuk-button--secondary">').text(settings.signOutButtonText))
+      .append($('<a id="timeout-sign-out-link" class="govuk-link hmrc-timeout-dialog__link">').text(settings.signOutButtonText))
```

Comment:
sign out link is not correctly visible ( should be blue and underline)

![image](https://user-images.githubusercontent.com/26648207/106725510-5d39c680-6601-11eb-9fa1-8b07a78854e8.png)

Reply:
signed-you-out is not getting called after timeout, its redirecting to normal sign signout